apps/src/music/views/MusicView.jsx

```diff
@@ -477,6 +512,11 @@ class UnconnectedMusicView extends React.Component {
       return;
     }
 
+    // Toolbox mode isn't intended to have a fully functional workspace,
+    // so we can skip the remaining logic for this event.
+    if (Blockly.isToolboxMode) {
+      return;
```

Comment:
Code below this point handles things like generating function blocks for a flyout toolbox, compiling the song, selecting blocks and saving code. None of these are relevant in toolbox mode, so it makes sense to skip them.